spec/address_book_spec.rb

```diff
@@ -55,35 +55,78 @@ def check_entry(entry, expected_name, expected_number, expected_email)
     it 'imports the 1st entry' do
       book.import_from_csv('entries.csv')
       entry_one = book.entries[0]
-      check_entry(entry_one, 'Bill', '555-555-4854', 'bill@blocmail.com')
+      check_entry(entry_one, 'Bill',
+                             '555-555-4854',
+                             'bill@blocmail.com')
     end
 
     it 'imports the 2nd entry' do
       book.import_from_csv('entries.csv')
       # Check the second entry
       entry_two = book.entries[1]
-      check_entry(entry_two, 'Bob', '555-555-5415', 'bob@blocmail.com')
+      check_entry(entry_two, 'Bob',
+                             '555-555-5415',
+                             'bob@blocmail.com')
     end
 
     it 'imports the 3rd entry' do
       book.import_from_csv('entries.csv')
       # Check the third entry
       entry_three = book.entries[2]
-      check_entry(entry_three, 'Joe', '555-555-3660', 'joe@blocmail.com')
+      check_entry(entry_three, 'Joe',
+                               '555-555-3660',
+                               'joe@blocmail.com')
     end
 
     it 'imports the 4th entry' do
       book.import_from_csv('entries.csv')
       # Check the fourth entry
       entry_four = book.entries[3]
-      check_entry(entry_four, 'Sally', '555-555-4646', 'sally@blocmail.com')
+      check_entry(entry_four, 'Sally',
+                              '555-555-4646',
+                              'sally@blocmail.com')
     end
 
     it 'imports the 5th entry' do
       book.import_from_csv('entries.csv')
       # Check the fifth entry
       entry_five = book.entries[4]
-      check_entry(entry_five, 'Sussie', '555-555-2036', 'sussie@blocmail.com')
+      check_entry(entry_five, 'Sussie',
+                              '555-555-2036',
+                              'sussie@blocmail.com')
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.